src/test/scala/tech/cryptonomic/conseil/tezos/ApiOperationsTest.scala

```diff
@@ -1,11 +1,112 @@
 package tech.cryptonomic.conseil.tezos
 
-import org.scalatest.{Matchers, WordSpec}
-import tech.cryptonomic.conseil.generic.chain.DataTypes.{OperationType, Predicate}
+import com.typesafe.scalalogging.LazyLogging
+import org.scalatest.concurrent.{IntegrationPatience, ScalaFutures}
+import org.scalatest.{Matchers, OptionValues, WordSpec}
+import tech.cryptonomic.conseil.generic.chain.DataTypes.{OperationType, Predicate, Query}
+import tech.cryptonomic.conseil.tezos.TezosTypes.{AccountId, BlockHash}
 
-class ApiOperationsTest extends WordSpec with Matchers {
+class ApiOperationsTest extends WordSpec
+  with Matchers
+  with InMemoryDatabase
+  with ScalaFutures
+  with OptionValues
+  with LazyLogging
+  with IntegrationPatience {
 
-  "ApiOperations" should {
+  import scala.concurrent.ExecutionContext.Implicits.global
+  "ApiOperationsTest" should {
+    val sut = new ApiOperations {
+      override lazy val dbReadHandle = dbHandler
+    }
+
+    "latestBlockIO" in {
+      // when
+      val result = dbHandler.run(sut.latestBlockIO()).futureValue
+
+      // then
+      result shouldBe None
+    }
+
+    "sanitizeForSql alphanumeric string" in {
+      // given
+      val input = "xyz123"
+
+      // when
+      val result = ApiOperations.sanitizeForSql(input)
+
+      // then
+      result shouldBe "xyz123"
+    }
+
+    "sanitizeForSql alphanumeric string with supported characters" in {
+      // given
+      val input = "xyz+123_abc: pqr"
+
+      // when
+      val result = ApiOperations.sanitizeForSql(input)
+
+      // then
+      result shouldBe "xyz+123_abc: pqr"
+    }
+
+    "sanitizeForSql alphanumeric string with unsupported characters" in {
+      // given
+      val input = ";xyz$%*)("
+
+      // when
+      val result = ApiOperations.sanitizeForSql(input)
+
+      // then
+      result shouldBe "xyz"
+    }
```

Comment:
Is it really necessary to use three separate tests instead of a single case containing good and bad chars and verifying that only the good ones are in?

Reply:
no, it's not necessary, but I think reading test output it's easier to understand what happened